src/Carbon/Carbon.php

```diff
@@ -343,12 +344,22 @@ public function __construct($time = null, $tz = null)
             $time = $testInstance->format(static::MOCK_DATETIME_FORMAT);
         }
 
+        // Get microseconds from microtime() if "now" asked and PHP < 7.1
+        $timezone = static::safeCreateDateTimeZone($tz);
+        // @codeCoverageIgnoreStart
+        if ($isNow && !isset($testInstance) && version_compare(PHP_VERSION, '7.1.0-dev', '<')) {
```

Comment:
Also add this for version 7.1.3 which always returns 0 of microseconds?

Reply:
No, PHP 7.1.3 works well, see the travis unit tests logs (and my locale PHP 7.1.3 works too), what trouble do you have?

Reply:
Microseconds should be also added to 7.1.3, since it is the only version of 7.1.x that doesn't return microseconds > https://3v4l.org/Re0Ul

Reply:
What's the heck? Give up to understand PHP changes., I will add that.

Reply:
Too bad, it means we will keep this in 2.x just to fix 7.1.3